.github/workflows/graalvm.yml

```diff
@@ -0,0 +1,76 @@
+# WARNING: Do not edit this file directly. Instead, go to:
+#
+# https://github.com/micronaut-projects/micronaut-project-template/tree/master/.github/workflows
+#
+# and edit them there. Note that it will be sync'ed to all the Micronaut repos
+name: GraalVM Dev CI
+on:
+  schedule:
+    - cron: "0 1 * * 1-5" # Mon-Fri at 1am UTC
+  workflow_dispatch:
+jobs:
+  build:
+    if: github.repository != 'micronaut-projects/micronaut-project-template'
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        graalvm: [ 'dev']
+        java: ['17']
```

Comment:
I'd use `dev` here, which should be future-proof. The combination `dev-17` may go away again